packages/ui/src/components/Typography/link.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import Base from 'antd/lib/typography/Base'
+import classNames from 'classnames'
+
+declare const TEXT_SIZES: ['lg', 'md', 'sm']
+export type TypographyLinkProps =  Omit<React.AnchorHTMLAttributes<HTMLAnchorElement>, 'type'> & {
+    title?: string
+    disabled?: boolean
+    ellipsis?: boolean
+    size?: typeof TEXT_SIZES[number]
+    id?: string
+    children?: React.ReactNode
+    ['aria-label']?: string
+}
+
+
+const TYPOGRAPHY_CLASS_PREFIX = 'condo-typography'
```

Comment:
Can this constant be shared across all components, related to typography?

Reply:
Refactored 👍